src/main.ts

```diff
@@ -59,13 +68,14 @@ async function main() {
     }
 }
 
-async function executeAzCliCommand(command: string, silent?: boolean) {
+async function executeAzCliCommand(command: string, silent?: boolean, options: any = {}) {
+    options.silent = !!silent;
     try {
-        await exec.exec(`"${azPath}" ${command}`, [],  {silent: !!silent}); 
+        await exec.exec(`"${azPath}" ${command}`, [],  options); 

```

Comment:
When doing az login, will this not show the service principal secret value ?

Reply:
no it will not show, because i am adding silent in options in previous line

Reply:
ok.